.travis.yml

```diff
@@ -2,9 +2,8 @@
 language: julia
 os:
   - linux
-  - osx
```

Comment:
Why drop Mac OS and Julia 0.5?

Reply:
I did in #108, so if this PR is rebased from master, that change will no longer show up in the diff. The Travis Mac queue is too long to be useful, especially since we don't have any platform-specific code in this package. Plus it was just time for 0.5 to be laid to rest so that we can fix the swarm of 0.7 deprecations.

Reply:
Change no longer shows up after rebase.